Makefile

```diff
@@ -14,6 +14,7 @@
 #
 # ESPRUINO_1V3=1          # Espruino board rev 1.3 and rev 1v4
 # PICO_1V3=1              # Espruino Pico board rev 1.3
+# PUCKJS=1                # Puck.js board
```

Comment:
And something if the no BOARD definition is found if one misspell the board name, like I have done a few times :)
But that means a lot of thought to do correct.
At the present time I usually do a quick grep and cut and paste the name